src/gausskernel/storage/access/k2/storage.h

```diff
@@ -47,6 +47,7 @@ struct K2PgScanHandle {
     std::deque<skv::http::dto::SKVRecord> queryRecords;
     std::deque<boost::future<skv::http::Response<skv::http::dto::SKVRecord>>> readReqs;
     K2PgSelectIndexParams indexParams;
+    uint32_t maxParallelReads = 5;
```

Comment:
do you still need this? I thought this is coming via config now

Reply:
I fill this in as part of newselect, so that the config doesn't need to be parsed for every fetch call.